### PR TITLE
Introduce BaseExecutor.validate_command to avoid duplication

### PR DESCRIPTION
**Pull request by turbaszek**
This PR proposes a simple method to avoid duplication of code in Executors.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/executors/celery_executor.py [hunk 1]
```diff
@@ -71,9 +71,7 @@
 @app.task
 def execute_command(command_to_exec: CommandType) -> None:
     """Executes command."""
-    if command_to_exec[0:3] != ["airflow", "tasks", "run"]:
-        raise ValueError('The command must start with ["airflow", "tasks", "run"].')
-
+    BaseExecutor.validate_command(command_to_exec)
```
Comment (kaxil) [inline airflow/executors/celery_executor.py:74]:
```suggestion
    cls.validate_command(command_to_exec)
```

that should work 🤷  - WDYT ?

Comment (turbaszek) [reply airflow/executors/celery_executor.py:74]: I think the `cls` will be undefined in this place

Comment (kaxil) [reply airflow/executors/celery_executor.py:74]: Oh yes, sry didnt realize it was not a part of the class

**Review by kaxil — APPROVED** [review]

File: airflow/executors/celery_executor.py (hunk 1 above)
Comment (turbaszek) [reply airflow/executors/celery_executor.py:74]: But I updated other usages to `self.` instead of `BaseExecutor.`

**Pull request merged**